packages/wrangler/src/cfetch/internal.ts

```diff
@@ -243,6 +243,12 @@ export function addUserAgent(headers: Headers): void {
 	headers.set("User-Agent", `wrangler/${wranglerVersion}`);
 }
 
+export function addTraceHeader(headers: Headers): void {
+	if (getTraceHeader()) {
+		headers.set("Cf-Trace-Id", getTraceHeader() as string);
+	}
```

Comment:
nit: avoid the cast and calling the fn twice:

```suggestion
	const traceHeader = getTraceHeader();
	if (traceHeader) {
		headers.set("Cf-Trace-Id", traceHeader);
	}
```